pandas/tests/extension/test_integer.py

```diff
@@ -0,0 +1,261 @@
+import numpy as np
+import pandas as pd
+import pandas.util.testing as tm
+import pytest
+
+from pandas.tests.extension import base
+from pandas.core.dtypes.common import is_extension_array_dtype
+
+from pandas.core.arrays import IntegerArray, integer_array
+from pandas.core.arrays.integer import (
+    Int8Dtype, Int16Dtype, Int32Dtype, Int64Dtype,
+    UInt8Dtype, UInt16Dtype, UInt32Dtype, UInt64Dtype)
+
+
+def make_data():
```

Comment:
This duplication with `pandas/tests/arrays` is unfortunate. I take it because they're in separate directories?

Reply:
Yes. I don't think it is worth importing it from one place to the other to remove the duplication for such a small thing.

But even when they are in the same directory, I personally think it is a good idea to keep all fixtures for the base extension tests separate from the ones we implement for the custom tests (because those last ones we might want to change regularly)

Reply:
BTW, I tried the approach of putting the extension interface test for integer also in `tests/arrays/integer` but marking them with a pytest mark so you can still easily run all interface tests for all types, but then you have the problem that the `tests/extension/conftest.py` is not used, and you would need to duplicate things from that ..

Reply:
I think duplicating fixtures is ok here (its not many)